sources/packages/backend/libs/integrations/src/services/msfaa-number/msfaa-number.service.ts

```diff
@@ -6,7 +6,11 @@ import {
   OfferingIntensity,
 } from "@sims/sims-db";
 import { getISODateOnlyString } from "@sims/utilities";
```

Comment:
Are we not suppose to send notification when we cancel an MSFAA during re-activation? 
![image](https://github.com/bcgov/SIMS/assets/54600590/ec7355a7-7379-44e5-a2e4-a29cf2a371bb)

Reply:
I would say no @dheepak-aot. In my understanding, we are deactivating some MSFAA(s) in order to activate other. The student will receive an email notification about it.
The point about sending a notification during the cancellation is that it may require an action from the student and it will not be communicated in any other way.